lib/ft817/ft817.cpp

```diff
@@ -29,7 +29,7 @@ This allow us to be consistent and save a few bytes of firmware
 #include "ft817.h"
 
 // define software serial IO pins here:
-extern SoftwareSerial rigCat(2, 3); // rx,tx
+extern SoftwareSerial rigCat(12, 11); // rx,tx

```

Comment:
Changed to match my FT817 Buddy PCB that I'm using for test against the radio

Reply:
no problem